cli/src/cli.rs

```diff
@@ -370,6 +350,16 @@ pub async fn new_wallet(cli: Cli) -> Result<Option<Wallet>, Error> {
     })
 }
 
+async fn print_create_wallet_summary(wallet: &Wallet, secret_manager_variant: &str) -> Result<(), Error> {
+    println_log_info!("Created new wallet with the following parameters:",);
+    println_log_info!("Secret manager: {secret_manager_variant}");
+    println_log_info!("Wallet address: {}", wallet.address().await);
+    println_log_info!("Wallet bip path: {:?}", wallet.bip_path().await);
+    println_log_info!("Wallet alias: {:?}", wallet.alias().await);
```

Comment:
No debug print needed?
```suggestion
    println_log_info!("Wallet alias: {}", wallet.alias().await);
```

Reply:
it's an `Option` what gets returned

Reply:
Ah true